airbyte-ci/connectors/connectors_qa/src/connectors_qa/checks/assets.py

```diff
@@ -19,14 +22,21 @@ def _run(self, connector: Connector) -> CheckResult:
             return self.create_check_result(
                 connector=connector,
                 passed=False,
-                message="Icon file is missing. Please create an icon file at the root of the connector code directory.",
+                message="Icon file is missing. Please create an icon file at the root of the connector code directory",
             )
         if not connector.icon_path.name == "icon.svg":
             return self.create_check_result(
                 connector=connector,
                 passed=False,
                 message="Icon file is not named 'icon.svg'",
             )
+        if connector.icon_path.read_text() == DEFAULT_AIRBYTE_ICON:
```

Comment:
```suggestion
        if connector.icon_path.read_text().strip().lower() == DEFAULT_AIRBYTE_ICON.strip().lower():
```

Reply:
I think we should handle exception that `read_text` might raise if the file is not text.